pupil_src/shared_modules/calibration_routines/fingertip_calibration/models/unet.py

```diff
@@ -64,8 +63,8 @@ def __init__(self, in_channels, out_channels, pooling=True):
             self.pool = nn.MaxPool2d(kernel_size=2, stride=2)
 
     def forward(self, x):
-        x = F.relu(self.conv1(x))
```

Comment:
I think it is not necessary to replace `F.relu` with `torch.relu`.

Reply:
You are right that it is not necessary. I did it for code consistency reasons. Do you know if there is any difference between `F.relu` and `torch.relu`?

Reply:
I just found this: https://github.com/pytorch/pytorch/blob/81e030d9a6e8d1d5c3d83628f37f2d49bbed597d/torch/nn/functional.py#L929
So then in our case, no difference between `F.relu` and `torch.relu`.

Reply:
Great find!